karma.conf.js

```diff
@@ -47,7 +47,8 @@ module.exports = function (config) {
     basePath: '',
     frameworks: ['mocha'],
     files: [
-      'test/tests.js'
+      'test/tests.js',
+      'test/constructor.js'
```

Comment:
I suggest to rename the files to all in end `.spec.js`, so we can do `test/*.spec.js`